### PR TITLE
Move get app version out of Core

### PR DESCRIPTION
**Pull request by AhmedAbdoOrtiga**
#### Describe the change
(Please provide an overview of the change in this PR)

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights/Modes/StartUpModeControl.xaml.cs [hunk 1]
```diff
@@ -77,7 +77,7 @@ protected override AutomationPeer OnCreateAutomationPeer()
 
         public StartUpModeControl()
         {
-            this.VersionString = Axe.Windows.Core.Misc.Utility.GetAppVersion();
+            this.VersionString =VersionTools.GetAppVersion();
```
Comment (DaveTryon) [inline src/AccessibilityInsights/Modes/StartUpModeControl.xaml.cs]: missing a space
```suggestion
            this.VersionString = VersionTools.GetAppVersion();
```

File: src/AccessibilityInsights.SharedUx/Misc/VersionTools.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace AccessibilityInsights.SharedUx.Misc
+{
+    public static class VersionTools
+    {
+        /// <summary>
+        /// Get version from AccessibilityInsights.SharedUx Assembly
+        /// </summary>
+        /// <returns></returns>
```
Comment (DaveTryon) [inline src/AccessibilityInsights.SharedUx/Misc/VersionTools.cs]: Add a comment here?
```suggestion
        /// <returns>The file version string (as formatted in the file resources)</returns>
```

**Comment by DaveTryon** [comment]
>         /// Check whether we need to show Splash window or not

Should be "Welcome", not "Splash" (we always display the splash screen)

---
Refers to: src/AccessibilityInsights.SharedUx/Settings/ConfigurationModel.cs:436 in 0557bc0. [](commit_id = 0557bc0764154d913beb59fe4e8f2f225eb65c44, deletion_comment = False)

File: src/AccessibilityInsights.SharedUx/Settings/ConfigurationModel.cs [hunk 3]
```diff
@@ -588,7 +589,7 @@ public static ConfigurationModel GetDefaultConfigurationModel()
         {
             ConfigurationModel config = new ConfigurationModel
             {
-                AppVersion = Axe.Windows.Core.Misc.Utility.GetAppVersion(),
+                AppVersion =VersionTools.GetAppVersion(),
```
Comment (DaveTryon) [inline src/AccessibilityInsights.SharedUx/Settings/ConfigurationModel.cs]: missing a space...
```suggestion
                AppVersion = VersionTools.GetAppVersion(),
```

**Comment by AhmedAbdoOrtiga** [comment]
> Check whether we need to show Splash window or not

This wasn't changed in the PR, but fixed!

**Pull request merged**